k8s/perftest/common/draft-store/draft-store-service.yaml

```
@@ -0,0 +1,45 @@
+---
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: draft-store-service
+  namespace: rpe
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: draft-store-service
+  chart:
+    repository: https://hmcts.azurecr.io/helm/v1/repo/
+    name: draft-store-service
+    version: 1.0.4
+  values:
+    rollback:
+      enable: true
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      applicationInsightsInstrumentKey: "88e9f6bc-d5ca-4bae-b408-03c271caa4a0"
+      image: hmctspublic.azurecr.io/draft-store/service:prod-6a5fa243
+      ingressHost: draft-store-service.service.core-compute-perftest.internal
+      environment:
+        DRAFT_STORE_DB_CONN_OPTIONS: ?sslmode=require
+        DRAFT_STORE_DB_HOST: rpe-draft-store-perftest.postgres.database.azure.com
+        DRAFT_STORE_DB_NAME: draftstore
+        DRAFT_STORE_DB_PORT: '5432'
+        DRAFT_STORE_DB_USER_NAME: draftstore@rpe-draft-store-perftest
+        RUN_DB_MIGRATION_ON_STARTUP: 'false'
+        IDAM_URL: https://idam-api.perftest.platform.hmcts.net
+        S2S_URL: http://rpe-service-auth-provider-perftest.service.core-compute-perftest.internal
+        SPRING_APPLICATION_NAME: "HMCTS Draft Store AKS"
+      aadIdentityName: rpe
+      keyVaults:
+        "draft-store":
+          resourceGroup: draft-store-service
+          secrets:
+          - service-POSTGRES-PASS
+    global:
+      environment: perftest
+      subscriptionId: "8a07fdcd-6abd-48b3-ad88-ff737a4b9e3c"
```

this should be for where the app team infra is:
```suggestion
      subscriptionId: "7a4e3bd5-ae3a-4d0c-b441-2188fee3ff1c"
```